requirements.txt

```diff
@@ -59,11 +59,11 @@ django-debug-toolbar==1.3.2
 django-waffle==0.10.1
 django-jenkins==0.17.0
 django-smoketest==0.3.0
-django-extensions==1.1.0
+django-extensions==1.5.2
 django-stagingcontext==0.1.0
 django-impersonate==0.9.1
 django-registration-redux==1.1
```

Comment:
I see. I should upgrade worth2 to 1.2. I think I added 1.2 to the wiki there because in the release notes it says that's the first version compatible with Django 1.8. Since this is working I'll merge it.